### ISSUE TITLE
Update Config table with the bids_vl

### ISSUE THREAD

**Issue by cmadjar** [aces/Loris#5429]
For some obscure reason, the following syntax made Travis unhappy:
```
-- Insert into Config tables
INSERT INTO ConfigSettings
  (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber)
  SELECT
    'default_bids_vl',
    'Default visit label to use when no visit label set in the BIDS dataset',
    1,
    0,
    'text',
    ID as parentID,
    'Default visit label for BIDS dataset',
    (SELECT MAX(OrderNumber) + 1 FROM ConfigSettings WHERE Parent=parentID)
  FROM ConfigSettings
  WHERE Name="imaging_pipeline";
```

Once #5425 is merged into major, send a PR to major to add that back into the Config schema tables.

**Comment by johnsaigle** [comment]
#5425 is merged
